src/OpenTelemetry.Instrumentation.Process/ProcessMetrics.cs

```diff
@@ -99,5 +142,26 @@ private void Snapshot()
         this.virtualMemoryUsage = this.currentProcess.PrivateMemorySize64;
         this.userProcessorTime = this.currentProcess.UserProcessorTime.TotalSeconds;
         this.privilegedProcessorTime = this.currentProcess.PrivilegedProcessorTime.TotalSeconds;
+        this.cpuUtilization = this.GetCpuUtilization();
+        this.numberOfThreads = this.currentProcess.Threads.Count;
+    }
+
+    private IEnumerable<Measurement<double>> GetCpuUtilization()
+    {
+        var userProcessorUtilization = (this.currentProcess.UserProcessorTime.TotalSeconds - this.collectionUserProcessorTime) /
```

Comment:
`this.collectionUserProcessorTime` and `this.collectionPrivilegedProcessorTime` will be zero for the first collection and would always result in a high value returned for first collection.

Reply:
I see. I updated to make `DateTime.UtcNow` the default for `lastCollectedTimeStampUtc`, which should fix this concern.